website_mrp_bom/README.rst

```diff
@@ -0,0 +1,21 @@
+**This file is going to be generated by oca-gen-addon-readme.**
```

Comment:
Don't you have to generate the README with [this script](https://github.com/OCA/maintainer-tools/blob/master/tools/gen_addon_readme.py)?

Reply:
nope, not mandatory